lib/Config/PopulatorData/Templates/Fitness.php

```diff
@@ -25,63 +25,66 @@ public function get() {
   }
 
   private function getThumbnail() {
-    return $this->template_image_url . '/thumbnail.20190411-1500.jpg';
+    return $this->template_image_url . '/fitness-thumbnail.20201028.jpg';
```

Comment:
We already have the template name as a folder in `$this->template_image_url`, so I think adding a name to the thumbnail file is redundant.